FAQ.md

```diff
@@ -355,3 +355,19 @@ The high availability of APISIX can be divided into two parts:
 1. The data plane of Apache APISIX is stateless and can be elastically scaled at will. Just add a layer of LB in front.
 
 2. The control plane of Apache APISIX relies on the highly available implementation of `etcd cluster` and does not require any relational database dependency.
+
+## Why does the `make deps` command fail in source installation?
+
+When executing the `make deps` command, an error such as the one shown below occurs. This is caused by the missing dependency of `openresty-openssl111-devel`, you need to install it first. Please refer to the [install-dependencies.md](doc/install-dependencies.md) document for installation.
```

Comment:
In Debian/Ubuntu Linux, the package name is not `openresty-openssl111-devel` but `openresty-openssl111-dev`. Here we don't need to show the real package name but tell people the development package of openssl for OpenResty is missing, guiding them to refer to the document.

Reply:
updated.